mathics/builtin/list/rearrange.py

```diff
@@ -432,7 +429,7 @@ def eval(self, lists, evaluation, options={}):
                 evaluation.message(
                     self.get_name(),
                     "normal",
-                    pos + 1,
+                    Integer(pos + 1),
```

Comment:
There are still a number of inconsistent type parameters in these message calls.